projects/plugins/jetpack/_inc/client/recommendations/sidebar/discount-card/index.jsx

```diff
@@ -54,26 +71,39 @@ const DiscountCard = ( { discount, expiryDate } ) => {
 							<li>{ __( 'Real-time malware scanning', 'jetpack' ) }</li>
 							<li>{ __( 'Comments and form spam protection', 'jetpack' ) }</li>
 						</ul>
-						<Button
-							className="jp-recommendations-discount-card__button"
-							rna
-							href={ '#/recommendations/product-suggestions' }
-							onClick={ onViewDiscountClick }
-						>
-							{ __( 'View discounted plans', 'jetpack' ) }
-						</Button>
+						{ ! ( isFetchingDiscount || isFetchingSuggestions || isFetchingOffers ) && (
+							<Button
+								className="jp-recommendations-discount-card__button"
+								rna
+								href={ '#/recommendations/product-suggestions' }
+								onClick={ onViewDiscountClick }
+							>
+								{ hasDiscount
+									? __( 'View discounted products', 'jetpack' )
+									: __( 'View products', 'jetpack' ) }
+							</Button>
+						) }
 					</div>
 				</div>
-				<div className="jp-recommendations-discount-card__timer">
-					<Timer
-						timeClassName="jp-recommendations-discount-card__time"
-						label={ __( 'Discount ends in:', 'jetpack' ) }
-						expiryDate={ expiryDate }
-					/>
-				</div>
+				{ hasDiscount && (
```

Comment:
Should we also check for `expiryDate` here?

Reply:
`hasDiscount` will be false is `expiryDate` is invalid or passed.